ms/MSOper/MSMetaData.h

```diff
@@ -85,12 +85,13 @@ class MSMetaData {
         // the subscan and that spwID
         map<uInt, Quantity> meanInterval;
         Quantity meanExposureTime;
-        uInt nrows;
         std::set<uInt> spws;
         // number of rows for each spectral window
         map<uInt, uInt> spwNRows;
         std::set<Int> stateIDs;
-        std::map<Double, TimeStampProperties> timeProps;
+        map<Double, TimeStampProperties> timeProps;
```

Comment:
the actual culprit is in some other header, but that is no reason to stick to bad style, also it avoids an error in for the case when the cause is fixed.